docs/platforms/flutter/index.mdx

```diff
@@ -37,11 +37,22 @@ Select which Sentry features you'd like to install in addition to Error Monitori
 
 Sentry captures data by using an SDK within your application's runtime. These are platform-specific and allow Sentry to have a deep understanding of how your application works.
 
-```yml {filename:pubspec.yaml}
-dependencies:
-  sentry_flutter: ^{{@inject packages.version('sentry.dart.flutter') }}
+To install, run `@sentry/wizard`:
+
+```bash
+brew install getsentry/tools/sentry-wizard && sentry-wizard -i flutter
```

Comment:
not sure whether to use brew or npx, I can imagine this won't be useful for people developing on windows

Reply:
or I can add a tab with both options

Reply:
tab sounds like a smart solution, I don't think we have a way to detect the user agent and e.g. automatically show npx for windows clients